syncer/syncer_test.go

```diff
@@ -400,29 +475,34 @@ func (s *testSyncerSuite) TestIgnoreTable(c *C) {
 		},
 	}
 
-	sqls := []string{
-		"create database s1",
-		"create table s1.log(id int)",
-		"drop database s1",
-
-		"create table mysql.test(id int)",
-		"drop table mysql.test",
-		"create database stest",
-		"create table stest.log(id int)",
-		"create table stest.t(id int)",
-		"create table stest.log2(id int)",
-		"insert into stest.t(id) values (10)",
-		"insert into stest.log(id) values (10)",
-		"insert into stest.log2(id) values (10)",
-		"drop table stest.log,stest.t,stest.log2",
-		"drop database stest",
-
-		"create database t2",
-		"create table t2.log(id int)",
-		"create table t2.log1(id int)",
-		"drop table t2.log",
-		"drop database t2",
-	}
+	s.resetEventsGenerator(c)
+	events := make([]mockBinlogEvent, 0, 12)
+	events = append(events, mockBinlogEvent{typ: DBCreate, args: []interface{}{"s1"}})
+	events = append(events, mockBinlogEvent{typ: TableCreate, args: []interface{}{"s1", "create table s1.log(id int)"}})
+	events = append(events, mockBinlogEvent{typ: DBDrop, args: []interface{}{"s1"}})
+	events = append(events, mockBinlogEvent{typ: TableCreate, args: []interface{}{"mysql", "create table mysql.test(id int)"}})
+	events = append(events, mockBinlogEvent{typ: TableDrop, args: []interface{}{"mysql", "test"}})
+
+	events = append(events, mockBinlogEvent{typ: DBCreate, args: []interface{}{"stest"}})
+	events = append(events, mockBinlogEvent{typ: TableCreate, args: []interface{}{"stest", "create table stest.log(id int)"}})
+	events = append(events, mockBinlogEvent{typ: TableCreate, args: []interface{}{"stest", "create table stest.t(id int)"}})
+	events = append(events, mockBinlogEvent{typ: TableCreate, args: []interface{}{"stest", "create table stest.log2(id int)"}})
+
+	events = append(events, mockBinlogEvent{typ: Write, args: []interface{}{uint64(8), "stest", "t", []byte{mysql.MYSQL_TYPE_LONG}, [][]interface{}{{int32(10)}}}})
+	events = append(events, mockBinlogEvent{typ: Write, args: []interface{}{uint64(9), "stest", "log", []byte{mysql.MYSQL_TYPE_LONG}, [][]interface{}{{int32(10)}}}})
+	events = append(events, mockBinlogEvent{typ: Write, args: []interface{}{uint64(10), "stest", "log2", []byte{mysql.MYSQL_TYPE_LONG}, [][]interface{}{{int32(10)}}}})
+	events = append(events, mockBinlogEvent{typ: TableDrop, args: []interface{}{"stest", "log"}})
```

Comment:
Currently, the binlog generator doesn't support generate an event with multiple tables DDL, shall we leave a TODO comment here.

Reply:
Done